### ISSUE TITLE
GeoJsonLayer doesn't use stroked and other properties in playground example

### ISSUE THREAD

**Issue by isaacbrodsky** [uber/deck.gl#4177]
#### Description
I was using the deck.gl playground and trying to disable some parts of the GeoJsonLayer, and found that they did not disable rendering those parts.

I expected that the example JSON below would disable rendering the lines (paths) around the polygons because `stroked` was set to `false`, and that it would disable rendering the polygons themselves because both `filled` was set to `false` and `opacity` was set to `0.0`. In fact, it renders both the stroke and the fill. `opacity` was respected when it was >0.

![deckgl playground stroked false](https://user-images.githubusercontent.com/9139378/72846925-63f3d680-3c56-11ea-9517-ea14be23f4f3.png)

#### Repro Steps
In the deck.gl playground, I tried to use the following example JSON:
```
{
  "description": "A deck.gl get-started GeoJsonLayer example in JSON format",
  "initialViewState": {
    "latitude": 40,
    "longitude": -100,
    "zoom": 3,
    "bearing": 30,
    "pitch": 30
  },
  "map": true,
  "views": [
    {
      "@@type": "MapView",
      "controller": true
    }
  ],
  "layers": [
    {
      "@@type": "GeoJsonLayer",
      "data": "https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_admin_1_states_provinces_shp.geojson",
      "stroked": false,
      "filled": false,
      "lineWidthMinPixels": 2,
      "opacity": 0.0,
      "getLineColor": [
        0,
        0,
        0
      ],
      "getFillColor": [
        200,
        160,
        0,
        180
      ]
    }
  ]
}
```

#### Environment (please complete the following information):
 - Framework Version: On the playground and confirmed locally with deck.gl@8.0.1
 - Browser Version: Firefox 72.0.1, also happens on a few versions of Chrome
 - OS: Linux (Ubuntu Bionic), and others

#### Logs
N/a

**Comment by ajduberstein** [comment]
Thanks for reporting this, @isaacbrodsky–it should be fixed by #4185.

**Comment by Pessimistress** [comment]
Fix published in 8.0.10
